### DISCUSSION TITLE
New Releases

### DISCUSSION THREAD

**Discussion by KevChief** [sct/overseerr#3948] [General]
Hi,

Just wanted to ask whether this is still an active repository and there will be new releases with new features and bug fixes?

It's been since 2023 , but I can see activity still in features requests and the community overall, so was wondering if anyone knows what the idea is here :)

**Comment by Lockszmith-GH** [comment]
I'm not a contributor, but from following this repo, I can tell you there are PRs that are actively been worked on.
There are commits from 2 weeks ago.

Definitely being worked on. As for release schedule - I'm not able to help - hopefully one of the contributors will replay.

**Reply by Lockszmith-GH** [reply]
Since I answered, things seem staler and staler. Jelliseerr seems to be where "it's at" - see my [reply below](https://github.com/sct/overseerr/discussions/3948#discussioncomment-12043381).

**Comment by aaronldunlap** [comment]
My seedbox only allows installing stable releases, so I’m now over a year behind development. It would be nice to have a new version release to finally stop having to manually deal with requests for deleted shows.

**Comment by carloss66** [comment]
This looks like a stale project. I think I will switch to Jellyseerr since development there is more active and it works with Plex, Emby, and Jellyfin.

**Reply by Ayymoss** [reply]
Look at the commit history, it's clearly not stale. Why would people be actively contributing if it was 'stale'?

**Reply by saber-oz** [reply]
I can see why people would think that.
The last release was 2023-07-28, so over a year ago.

Been lots of commits but no releases so from a release point of view you can see why it might be seen as stale.

Love this software though.

**Reply by Ayymoss** [reply]
> Been lots of commits but no releases so from a release point of view you can see why it might be seen as stale.

They would be wrong. It's actively being developed. It'd be a wasted effort if no release comes from this - logically follows it is not stale.

**Reply by saber-oz** [reply]
I am sure it is but releases are what showcases development and it is what people can see.
logical flow shouldn't see a 16month gap between release - why would someone want to do commits if it will take over a see to see the work.

you say it would be a wasted effort but with no release so far it is wasted because dev's hard work is not being appreciated by the community

**Reply by KevChief** [reply]
So how to proceed ? Is there something who is able to create such a new release of the current development status?

**Reply by Ayymoss** [reply]
If you're looking for a specific feature, you may want to build it yourself. Otherwise, you'll need to be patient and wait for an official release build.

**Reply by Lockszmith-GH** [reply]
Jellyseerr is where the development seems to continue - it supports plex as a backend for some time now.
Responding here, as I've finally installed Jellyseerr yesterday, and looks like q few PRs I've seen active here, have been implemented over there.

I suggest focus be shifter there.
